server/src/db/schema.rs

```diff
@@ -0,0 +1,24 @@
+table! {
+    node (id) {
+        id -> Int4,
+        uid -> Uuid,
+    }
+}
+
+table! {
+    person (id) {
+        id -> Int4,
+        node_id -> Int4,
+        display_name -> Varchar,
+        email -> Varchar,
+        first_name -> Varchar,
+        last_name -> Varchar,
```

Comment:
Question for my learning:

When we need to add columns to describe a person, we can use up.sql and down.sql for migration. How often do we expect to do a migration? Are we going to do it manually only?

Reply:
In fact this file is automatically generated by migrations, so the only way we should be manually modifying it is by adding a migration and running it. (I just found out it's possible to configure the Diesel CLI to add doc comments showing that this is generated code, I'll enable that)

After we define the core set of objects in the database, we probably will only rarely need to do migrations.